samples/drivers/led_ws2812/boards/nucleo_h563zi.overlay

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022, STMicroelectronics
+ * Copyright (c) 2024, Vitrolife A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/led/led.h>
+
+&arduino_spi {
+	led_strip: ws2812@0 {
+		compatible = "worldsemi,ws2812-spi";
+
+		/* SPI */
+		reg = <0>; /* ignored, but necessary for SPI bindings */
+		spi-max-frequency = <DT_FREQ_M(4)>;
+		frame-format = <32768>; /* SPI_FRAME_FORMAT_TI */
+
+		/* WS2812 */
+		chain-length = <720>; /* arbitrary; change at will */
```

Comment:
720 is a bit much, 16 is used by other boards

Reply:
Excellent ! Does it work ? :)

Reply:
Turns out the LEDs are in culsters of 15 LEDs, so it is 720 LEDs / 15 = 48.
I've updated the code.